### ISSUE TITLE
Webcam qr-code reader crash if no qr provided

### ISSUE THREAD

**Issue by zveda** [fyookball/electrum#324]
Am in Linux using electron-cash 3.0. When using the webcam zbarcam qr code reader, if I just close the window without giving a QR code, I get:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/Electron_Cash-3.0-py3.5.egg/electroncash_gui/qt/paytoedit.py", line 263, in qr_input
    if data.startswith("bitcoincash:"):
AttributeError: 'NoneType' object has no attribute 'startswith'
Aborted
```
And electron-cash just crashes and exits. If I give it a qr-code to read, then it works fine.

**Comment by hegjon** [comment]
I think this also happens in Electrum, but it does not crash the whole application

**Comment by zveda** [comment]
Just tested it and get the same thing in electrum:
`Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/electrum_gui/qt/paytoedit.py", line 263, in qr_input
    if data.startswith("bitcoin:"):
AttributeError: 'NoneType' object has no attribute 'startswith'
Aborted`
and electrum crashes and exits

**Comment by marceloneil** [comment]
Can you try this with electrum 3.1.3 or electron-cash 3.2?

**Comment by hegjon** [comment]
I confirm that this issue have been fixed in 3.2
